src/main/java/net/dries007/tfc/objects/items/ItemOverriddenGlassBottle.java

```diff
@@ -0,0 +1,89 @@
+/*
+ * Work under Copyright. Licensed under the EUPL.
+ * See the project README.md and LICENSE.txt for more information.
+ */
+
+package net.dries007.tfc.objects.items;
+
+import java.util.Optional;
+import javax.annotation.ParametersAreNonnullByDefault;
+
+import net.minecraft.block.material.Material;
+import net.minecraft.block.state.IBlockState;
+import net.minecraft.entity.player.EntityPlayer;
+import net.minecraft.init.Blocks;
+import net.minecraft.init.Items;
+import net.minecraft.item.ItemGlassBottle;
+import net.minecraft.item.ItemStack;
+import net.minecraft.util.ActionResult;
+import net.minecraft.util.EnumActionResult;
+import net.minecraft.util.EnumHand;
+import net.minecraft.util.math.RayTraceResult;
+import net.minecraft.world.World;
+import net.minecraftforge.fluids.FluidActionResult;
+import net.minecraftforge.fluids.FluidUtil;
+import net.minecraftforge.fluids.capability.IFluidHandler;
+import net.minecraftforge.fluids.capability.IFluidTankProperties;
+import net.minecraftforge.items.IItemHandler;
+
+import mcp.MethodsReturnNonnullByDefault;
+import net.dries007.tfc.util.Helpers;
+
+import static net.minecraftforge.items.CapabilityItemHandler.ITEM_HANDLER_CAPABILITY;
+
+@MethodsReturnNonnullByDefault
+@ParametersAreNonnullByDefault
+
+public class ItemOverriddenGlassBottle extends ItemGlassBottle
```

Comment:
Keeping consistent with our other registry overrides, this should be called `ItemGlassBottleTFC`

Reply:
renamed